packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -842,21 +861,43 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   void _cursorTick(Timer timer) {
     _showCursor.value = !_showCursor.value;
+
+    if (_platformIsIOS) {
+      final double toValue = _showCursor.value ? 1.0 : 0.0;
+      _cursorController.animateTo(toValue, duration: _fadeDuration, curve: Curves.easeOut);
```

Comment:
Describe your methodology (as to inform the next maintainer's confidence when they look at this and decide how much scrutiny they should go through first before changing it).

Reply:
If _fadeDuration is const, I think you can just pass it into the animationcontroller and not specify it here each time.